### ISSUE TITLE
Merge Base Names where original is missing

### ISSUE THREAD

**Issue by thely** [ChrisHinde/MaterialUtilities#6]
Let's say I import an object into my scene that has a material. I decide I want to copy it into another scene, but I forget to add it as a duplicate, so Material is now Material.001. I copy it back into the original scene, again failing to add as a duplicate, so now I have Material, Material.001, and Material.002. I delete the original object; Material is now gone, though .001 and .002 remain.

Basically: is there a way to adapt Merge Base Names to also search for materials that are missing their original? I unfortunately have a lot of examples of the above problem, and Merge Base Names misses all these stragglers.

**Comment by ChrisHinde** [comment]
Previous versions could Merge materials to a single Material without a material with a "base name" like "Material" (if you selected "Material.001", it would rename it to "Material" and *merge* "Material.002" etc with it.

This now also works with "Auto Rename/Replace" in the latest version: If it finds a material named "Wood.001", but no material named "Wood", it will rename "Wood.001" and use that *merge* all matching materials to that.
(This would've produced an error in previous versions)
